<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1481</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Метод Сопряжённых Градиентов, Щербаков Павел, гр. 3821Б1ПМоп2, Вар. 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>931</width>
      <height>321</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(113, 113, 113);</string>
    </property>
    <layout class="QGridLayout" name="information_layout" columnstretch="1,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ввод параметров</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="1">
          <widget class="QLineEdit" name="Nmax2"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>eps_met, параметр выхода по точности (тест. зад. и осн. на гл. сетке)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Nmax2, макс. кол-во итераций (осн. зад. на контрольной сетке)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="task_type">
           <property name="currentText">
            <string>Тестовая</string>
           </property>
           <item>
            <property name="text">
             <string>Тестовая</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Основная</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="eps_met"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Задача</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="eps_met2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>eps_met2, параметр выхода по точности (осн. зад. на контрольной сетке)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>m, количество разбиений по y</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;количество разбиений по x&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>n, количество разбиений по x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLineEdit" name="input_n"/>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Nmax"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="input_m"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Nmax, макс. кол-во итераций (тест. задача и осн. на главной сетке)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Численное решение</string>
     </attribute>
     <widget class="QWidget" name="plot_widget_1" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>911</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(148, 148, 148);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Начальное приближение</string>
     </attribute>
     <widget class="QWidget" name="plot_widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>911</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(148, 148, 148);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Таблицы</string>
     </attribute>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>531</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>901</width>
         <height>396</height>
        </rect>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="label">
        <string>Точное решение</string>
       </attribute>
       <widget class="QTableWidget" name="info_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>401</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>9999999</height>
         </size>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <column>
         <property name="text">
          <string>i</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Xi</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Vi</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>V'i</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>V2i</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Vi-V2i</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ОЛП</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>h_i</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>C1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>C2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ui</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>|Ui-Vi|</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>901</width>
         <height>396</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Численное решение</string>
       </attribute>
       <widget class="QTableWidget" name="info_table_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>901</width>
         <height>396</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Разность точного и числ решения</string>
       </attribute>
       <widget class="QTableWidget" name="info_table_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>901</width>
         <height>396</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Начальное приближение для осн. сетки</string>
       </attribute>
       <widget class="QTableWidget" name="info_table_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>901</width>
         <height>396</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Начальное приближение на контрольной сетке</string>
       </attribute>
       <widget class="QTableWidget" name="info_table_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>841</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="plot_button">
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="delete_plot">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>очистить</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>80</y>
      <width>531</width>
      <height>851</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="extra_info_layout"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>301</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>50</y>
      <width>153</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Выходные данные программы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="info_button"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="info_button">
   <property name="text">
    <string>Описание</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>input_n</tabstop>
  <tabstop>input_m</tabstop>
  <tabstop>Nmax</tabstop>
  <tabstop>eps_met</tabstop>
  <tabstop>Nmax2</tabstop>
  <tabstop>task_type</tabstop>
  <tabstop>plot_button</tabstop>
  <tabstop>delete_plot</tabstop>
  <tabstop>info_table_2</tabstop>
  <tabstop>info_table</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
